apps/livre/livre.cpp

```diff
@@ -59,6 +60,7 @@ int main( const int argc, char** argv )
     {
         if( std::string( "--help" ) == argv[i] )
         {
+            livre::DataSource::loadPlugins(); // needed to complete --volume
```

Comment:
hide in Client::getHelp(). Or leave it, as it will conflict with my upcoming changes